packages/pytest-simcore/src/pytest_simcore/postgres_service.py

```diff
@@ -38,15 +38,10 @@ def postgres_dsn(docker_stack: Dict, devel_environ: Dict) -> Dict[str, str]:
 @pytest.fixture(scope="module")
 def postgres_db(postgres_dsn: Dict[str, str], docker_stack: Dict) -> sa.engine.Engine:
     url = DSN.format(**postgres_dsn)
-
     # Attempts until responsive

```

Comment:
**IMPORTANT**: I have the impression that you have undone everything during a merge from master:

See the original in master for this piece https://github.com/ITISFoundation/osparc-simcore/blob/master/packages/pytest-simcore/src/pytest_simcore/postgres_service.py#L42

Reply:
funny cause I think you actually told me Retrying is not working asynchronously

Reply:
and I changed them according to that fact

Reply:
but this is synchronous ... anyway, the Retry issue is now clean :-)